proto/buf/alpha/registry/webhook/v1alpha1/event_payloads.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.webhook.v1alpha1;
```

Comment:
See above

Reply:
Moved back to parent pkg.

Reply:
I wasn't necessarily implying parent package, I meant that we don't have parent/child packages, ie the current standard would say that if you are creating a new package, it would be `buf.alpha.webhook.v1alpha1`.

Reply:
But if we do that, then `WebhookEvent` and `WebhookRequest` would live in 2 different packages, since `WebhookEvent` is used when subscribing to an event.

Reply:
Or they would both live in a `buf.alpha.webhook.v1alpha1` package, alongside the service users are expected to implement, which is what I was hinting at (but could have been more explicit), also re: https://github.com/bufbuild/buf/pull/1249#discussion_r911236908=

Reply:
We could move everything webhook-related to a separate `webhook` package, but I think it belongs more to `registry`:

- Create/Delete/List webhooks is something that's implemented by the BSR UI
- The BSR implements the client stub of the webhook event service (customer would be implementing the server stub)
- The payload of the event contains (and will contain as we keep adding more events) several nested messages that already live in the registry package (repository, commits, users...)

If we separate to a different package, we could still use/import those messages from the registry, but I think it'd be better to declare a new "pubic" repository message, to be embedded in the webhook payload message.